msbuild/domoticz.vcxproj.filters

```diff
@@ -438,6 +438,8 @@
     <Filter Include="Devices\HEOS">
       <UniqueIdentifier>{43faa747-76f8-47a8-a90b-ebcb30a1f72e}</UniqueIdentifier>
     </Filter>
+    <Filter Include="Devices\MultiFun">
+      <UniqueIdentifier>{61f9e7d0-4204-474d-b953-b8d55b281296}</UniqueIdentifier>
```

Comment:
Did you open the visual studio project ... in visual studio ?
It was broken.... for example you missed the closing tag (</Filter)
Also below for include/compile